backend/storage/storage.go

```diff
@@ -54,6 +54,7 @@ var (
 const (
 	MIME_TYPE_JSON          = "application/json"
 	CONTENT_ENCODING_BROTLI = "br"
+	MAX_DOWNLOAD_SIZE       = 10 * 1024 * 1024 // 10MB
```

Comment:
makes sense to do this for live sessions, though this might be a bit of a low limit. wonder if we should do something in the UI to inform the user of why the live mode session might be not showing the network?